cereal/custom.capnp

```diff
@@ -78,6 +78,11 @@ struct ModelManagerSP @0xaedffd8f31e7b55d {
     stock @2;
   }
 
+  struct Overrides {
+    lat @0 :Float32;
+    long @1 :Float32;
+  }
+
```

Comment:
instead of us hardcoing the potential overrides, I had in mind that it would be more a `map(key, value)` similar to how the params are pushed on init